strings/src/main/java/com/revenuecat/purchases/strings/ReceiptStrings.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchases.strings
+
+object ReceiptStrings {
+    const val DATE_STALE_BACKGROUND = "Checking if cache is stale AppInBackground %s"
```

Comment:
this isn't tied to a date at all, maybe `CHECKING_IF_CACHE_STALE_BACKGROUND` would be more appropriate